Pepperdash Core/Pepperdash Core/Comm/GenericSshClient.cs

```diff
@@ -518,10 +543,21 @@ public void SendBytes(byte[] bytes)
                 {
                     Debug.Console(1, this, "Client is null or disconnected.  Cannot Send Bytes");
                 }
-			}
-			catch
-			{
-				Debug.Console(1, this, Debug.ErrorLogLevel.Error, "Stream write failed. Disconnected, closing");
+            }
+            catch (ObjectDisposedException ex)
+            {
+                Debug.Console(0, this, "Exception: {0}", ex.Message);

```

Comment:
Same thing here. Send the messages to the error log, but only send the message to console at 0, stack trace at 1